packages/front-generator/src/generators/react-typescript/app/params.ts

```diff
@@ -0,0 +1,27 @@
+import {StudioTemplateProperty, StudioTemplatePropertyType} from "../../../common/studio/studio-model";
+import {StudioProjectInfo} from "../../../common/studio/studio-integration";
+
+export const appGeneratorParams: StudioTemplateProperty[] = [
+  {
+    code: 'restClientId',
+    caption: 'Rest client id',
```

Comment:
`REST Client ID`

I know it may look like nitpicking, but I believe proper capitalization and stuff like that is something that can really affect the user's impression. I believe we should be really strict about this sort of thing.